<head>
  <style>
    .log { font-family: "Courier New", Consolas, monospace; font-size: 80%; }
    .test-failed { background-color: #ffaaaa; }
    .test-passed { background-color: #aaffaa; }
  </style>
</head>
<body>
<h3><a href="http://llhd.io/" target="_blank">moore</a></h3>
<pre class="test-failed">
description: Test imported from ivtest
rc: 101 (means success: 0)
tags: ivtest
incdirs: /tmpfs/src/github/sv-tests/third_party/tests/ivtest /tmpfs/src/github/sv-tests/third_party/tests/ivtest/ivltests
top_module: 
type: parsing
mode: parsing
files: <a href="../../../../third_party/tests/ivtest/ivltests/pr2350934.v.html" target="file-frame">third_party/tests/ivtest/ivltests/pr2350934.v</a>
defines: 
time_elapsed: 0.008s
ram usage: 11152 KB
</pre>
<pre class="log">

moore -I /tmpfs/src/github/sv-tests/third_party/tests/ivtest -I /tmpfs/src/github/sv-tests/third_party/tests/ivtest/ivltests -e test <a href="../../../../third_party/tests/ivtest/ivltests/pr2350934.v.html" target="file-frame">third_party/tests/ivtest/ivltests/pr2350934.v</a>
error: `block` not found
  --&gt; <a href="../../../../third_party/tests/ivtest/ivltests/pr2350934.v.html#l-5" target="file-frame">third_party/tests/ivtest/ivltests/pr2350934.v:5</a>:21-26:
   | 
   |     initial dummy = block.f(0);
   |                     ^^^^^      

warning: `block.f(0)` is not something that can be called
  --&gt; <a href="../../../../third_party/tests/ivtest/ivltests/pr2350934.v.html#l-5" target="file-frame">third_party/tests/ivtest/ivltests/pr2350934.v:5</a>:21-31:
   | 
   |     initial dummy = block.f(0);
   |                     ^^^^^^^^^^ 

compiler bug: type assertion failed: `&lt;error&gt;` != `reg [2:0]`
  --&gt; <a href="../../../../third_party/tests/ivtest/ivltests/pr2350934.v.html#l-5" target="file-frame">third_party/tests/ivtest/ivltests/pr2350934.v:5</a>:21-31:
   | 
   |     initial dummy = block.f(0);
   |                     ^^^^^^^^^^ 
   = note: Type mismatch:
   = note:     Left-hand side:  `&lt;error&gt;`
   = note:     Right-hand side: `reg [2:0]`
   = note: Encountered at src/svlog/codegen.rs:2003

You have encountered a compiler bug. Sorry about that! We would appreciate if you open an issue [1] and describe how you triggered the bug, together with a minimal snippet of code to reproduce it. Thanks!
[1]: https://github.com/fabianschuiki/moore

thread &#39;main&#39; panicked at &#39;type assertion failed: `&lt;error&gt;` != `reg [2:0]`&#39;, /tmp/really-really-really-really-really-really-really-really-really-really-really-really-really-long-path/conda/conda-bld/moore_1602170488679/work/src/svlog/codegen.rs:2003:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

</pre>
</body>